backend/core/detection.py

```diff
@@ -48,12 +48,14 @@
     ["S", "0", "-", "L"],
     ["S", "1", "-", "-"],
     ["S", "1", "-", "L"],
+    ["S", "P", "-", "-"],
```

Comment:
Can you also add the new hijack type on the documentation?

Reply:
Added in PR (`basicconf.md` + `hijackinfo.md`). Resolving.